source/MaterialXTest/MaterialXCore/Node.cpp

```diff
@@ -574,6 +574,55 @@ TEST_CASE("Organization", "[nodegraph]")
     CHECK(nodeGraph->getBackdrops().empty());
 }
 
+TEST_CASE("Tokens", "[nodegraph]")
+{
+    mx::DocumentPtr doc = mx::createDocument();
+    mx::loadLibrary(mx::FilePath::getCurrentPath() / mx::FilePath("libraries/stdlib/stdlib_defs.mtlx"), doc);
+    mx::loadLibrary(mx::FilePath::getCurrentPath() / mx::FilePath("libraries/stdlib/stdlib_ng.mtlx"), doc);
+    mx::FileSearchPath searchPath("resources/Materials/TestSuite/stdlib/texture/");
+
+    mx::readFromXmlFile(doc, "tokenGraph.mtlx", searchPath);
+    doc->validate();
+
+    mx::NodeGraphPtr graph = doc->getNodeGraph("Tokenized_Image");
+    if (graph)
```

Comment:
Nitpick: Maybe call `REQUIRE(graph)` here instead, since the test can't run if the test graph cannot be found.